dynamics-nav/SETFILTER-Function--Record-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETFILTER Function (Record)"
+description: SETFILTER Function (Record) Assigns a filter to a field that you specify.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.